Flaky Performance Test: Usage 55.19% is less then expected 56.00%

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://kubernetes.io/security/
-->


**What happened**:
```
{Failed  === RUN   TestScalability/ClusterQueueClasses/cq
    checker_test.go:103: Usage 55.19% is less then expected 56.00%
--- FAIL: TestScalability/ClusterQueueClasses/cq (0.00s)
}
```

**What you expected to happen**:
Errors never has been seen.

**How to reproduce it (as minimally and precisely as possible)**:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/2636/pull-kueue-test-scheduling-perf-main/1813914677910966272

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Kueue version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/test/performance/scheduler/checker/checker_test.go
+++ b/test/performance/scheduler/checker/checker_test.go
@@ -92,13 +92,13 @@ func TestScalability(t *testing.T) {
 	})
 
 	t.Run("ClusterQueueClasses", func(t *testing.T) {
-		for c, cqcSummarry := range summary.ClusterQueueClasses {
+		for c, cqcSummary := range summary.ClusterQueueClasses {
 			t.Run(c, func(t *testing.T) {
 				expected, found := rangeSpec.ClusterQueueClassesMinUsage[c]
 				if !found {
 					t.Fatalf("Unexpected class")
 				}
-				actual := float64(cqcSummarry.CPUUsed) * 100 / (float64(cqcSummarry.NominalQuota) * float64(cqcSummarry.LastEventTime.Sub(cqcSummarry.FirstEventTime).Milliseconds()))
+				actual := float64(cqcSummary.CPUUsed) * 100 / (float64(cqcSummary.NominalQuota) * float64(cqcSummary.LastEventTime.Sub(cqcSummary.FirstEventTime).Milliseconds()))
 				if actual < expected {
 					t.Errorf("Usage %.2f%% is less then expected %.2f%%", actual, expected)
 				}

--- a/test/performance/scheduler/default_rangespec.yaml
+++ b/test/performance/scheduler/default_rangespec.yaml
@@ -15,8 +15,8 @@ cmd:
   maxrss: 468_000
 
 clusterQueueClassesMinUsage:
-  # Average value 58.7 (+/- 1.2%), setting at -5%
-  cq: 56 #%
+  # Average value 58.7 (+/- 1.2%), setting at -7%
+  cq: 55 #%
 
 wlClassesMaxAvgTimeToAdmissionMs:
   # Average value 6666 (+/- 14%), setting at +35% + 1s accounting for observed failures

